Automattic/wp-calypso
# Page Pattern: Render patterns on client side

#### Proposed Changes

* Render page patterns on the ETK plugin and get rid of the preview endpoint and mshot

See p1666257227083559-slack-CRWCHQGUB for discussions.

https://user-images.githubusercontent.com/13596067/199210334-b31e66e2-f737-423a-a335-ca9115147317.mov

#### Testing Instructions



* Sandbox both the API and your site 
* Run `cd apps/editing-toolkit && yarn dev --sync`. Or run `install-plugin.sh editing-toolkit feat/render-page-patterns-on-client-side` on your sandbox.
* Go to your site and create a new page
* Verify the site title, tagline and logo are rendered correctly

#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/69555
